paddle/cinn/hlir/framework/op_lowering.cc

```diff
@@ -70,532 +81,332 @@ std::vector<ir::LoweredFunc> OpLowerer::Lower(GroupPtr& group) {  // NOLINT
   }
 }
 
-std::vector<ir::LoweredFunc> OpLowerer::LowerWithoutSchedule(GroupPtr& group) {
-  VLOG(3) << "Lowering Group : " << group->group_id
-          << " , Op Pattern : " << group->op_pattern_kind;
-  if (FLAGS_cinn_ir_schedule) {
-    switch (group->op_pattern_kind) {
-      case framework::kElementWise:
-      case framework::kBroadcast:
-      case framework::kInjective:
-        return IRLowerOpWithoutSchedule(&OpLowerer::IRElementwiseCompute,
-                                        group);
-      case framework::kReduction:
-        return IRLowerOpWithoutSchedule(&OpLowerer::IRReduceCompute, group);
-      case framework::kOutFusible:
-        LOG(FATAL) << "Group Pattern Kind kOutFusible Is Not Implemented!";
-      case framework::kNonFusible:
-        return IRLowerNonFusibleOp(group, /*apply_impl_schedule = */ false);
-      default:
-        LOG(FATAL) << "Group Pattern Kind kNonFusible Is Not Implemented!";
-    }
-  } else {
-    LOG(FATAL) << "Previous IR Schedule Is Not Implemented!";
+bool OpLowerer::ElementwiseScheduleDetermineFunction(Node* node) {
+  return true;
+}
+
+bool OpLowerer::ReduceScheduleDetermineFunction(Node* node) {
+  auto& op_pattern_dict = Operator::GetAttrs<OpPatternKind>("OpPattern");
+  if (op_pattern_dict[node->op()] == framework::kReduction) {
```

Comment:
Rewrite it:
`return op_pattern_dict[node->op()] == framework::kReduction`;

Reply:
Done.